projects/lib-posts/src/lib/lib-posts.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { LibPostsComponent } from './lib-posts.component';
+
+describe('LibPostsComponent', () => {
+  let component: LibPostsComponent;
```

Comment:
use const

Reply:
If I use const, then it must be initialized initially, and if it is initialized initially as const, then I cannot change it every time in beforeEach

Reply:
ok